Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1

```diff
@@ -96,7 +96,7 @@ try
         Write-Verbose "Environment details $environment"
 
         Write-Verbose "Calling Deploy test agent cmdlet"
-        Invoke-DeployTestAgent -TaskContext $distributedTaskContext -MachineEnvironment $environment -UserName $machineUserName -Password $machinePassword -MachineNames $testMachineGroup -RunAsProcess $runAsProcess -LogonAutomatically $logonAutomatically -DisableScreenSaver $disableScreenSaver -AgentLocation $agentLocation -UpdateTestAgent $updateTestAgent -InstallAgentScriptLocation $installAgentScriptLocation -ConfigureTestAgentScriptLocation $configureTestAgentScriptLocation -CheckAgentInstallationScriptLocation $checkAgentInstallationScriptLocation -downloadTestAgentScriptLocation $downloadTestAgentScriptLocation -Connection $connection -PersonalAccessToken $personalAccessToken -DataCollectionOnly $isDataCollectionOnly -VerifyTestMachinesAreInUseScriptLocation $null
```

Comment:
We thought about this last sprint and decided not to do this, since we will be forcing the users on older agent to upgrade unnecessarily.